cmd/dbseed/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/appconfig"
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+	"github.com/cmsgov/mint-app/pkg/models"
+	"github.com/cmsgov/mint-app/pkg/storage"
+
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+)
+
+// The main entrypoint for the dbseed command.
+// Invoke with "go run cmd/dbseed"
+func main() {
+	var rootCmd = &cobra.Command{
+		Use:   "dbseed",
+		Short: "Seed the DB",
+		Long:  "Seeds the Database with Model Plans and associated data",
+		Run: func(cmd *cobra.Command, args []string) {
+			config := viper.New()
+			config.AutomaticEnv()
+			seedData(config)
+		},
+	}
+
+	err := rootCmd.Execute()
+	if err != nil {
+		panic(err)
+	}
+}
+
+// getResolverDependencies takes a Viper config and returns a Store and Logger object to be used
+// by various resolver functions.
+func getResolverDependencies(config *viper.Viper) (*storage.Store, *zap.Logger) {
+	logger, err := zap.NewProduction()
```

Comment:
@ClayBenson94 , what do you think about this being a NOP logger?  The only thing I'm noticing is that it logs a lot of unneeded information to the console when you run the db:seed command.  That's not really the end of the world though, and I guess we could swallow the logging in the scripts/dev db:seed.  I don't really know if it's the end of the world to have the logging, and it might be useful. I also wonder how hard it would be to set the logging level here to just error or greater?

![image](https://user-images.githubusercontent.com/99294987/190220193-cdc6b558-dda2-420d-a92f-dc6b4ec4e496.png)

Reply:
I like this. Changing!